enzy_htp/structure/ligand.py

```diff
@@ -35,15 +35,38 @@ class Ligand(Residue):
 
     Attributes:
         net_charge : The net charge of the molecule as an int.
+        bonds: 
+        conformer_coords:
     """
 
     def __init__(self, residue_idx: int, residue_name: str, atoms: List[Atom], parent=None, **kwargs):
         """
         Constructor for Ligand. Identical to Residue() ctor but also takes net_charge value.
         """
+        self.bonds = kwargs.get('bonds', list()) 
```

Comment:
looks good for now. I might change it later during development of initiating connectivity for all kinds of Structure parts.
I'm thinking about have a init_connectivity method in Ligand that can have several different external software engines. And it will assign connectivity to Atom()s. And I will make self.bonds a property that returns the same format but deduced from information stored in Atom()s

Reply:
[No change needed]